src/main/java/frc/robot/subsystems/swerve/modules/factory/MK4IModuleContainer.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot.subsystems.swerve.modules.factory;
+
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.ctre.phoenix6.signals.FeedbackSensorSourceValue;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.ctre.phoenix6.signals.SensorDirectionValue;
+import frc.robot.subsystems.swerve.modules.ModuleID;
+import frc.robot.subsystems.swerve.modules.mk4i.MK4IModuleConstants;
+
+public class MK4IModuleContainer {
+
+    private static final double DRIVE_SLIP_CURRENT = 30; //todo - calibrate
+    private static final double STEER_CURRENT_LIMIT = 30; //todo - calibrate
+
+    private static final CANcoderConfiguration ENCODER_CONFIG = new CANcoderConfiguration();
+    static {
+        ENCODER_CONFIG.MagnetSensor.SensorDirection = SensorDirectionValue.CounterClockwise_Positive;
+        ENCODER_CONFIG.MagnetSensor.AbsoluteSensorRange = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    }
+
+    private static final TalonFXConfiguration DRIVE_MOTOR_CONFIG = new TalonFXConfiguration();
+    static {
+        DRIVE_MOTOR_CONFIG.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+        DRIVE_MOTOR_CONFIG.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+        DRIVE_MOTOR_CONFIG.Feedback.SensorToMechanismRatio = 6.12;
+
+        DRIVE_MOTOR_CONFIG.TorqueCurrent.PeakForwardTorqueCurrent = DRIVE_SLIP_CURRENT;
```

Comment:
torque current and slip current are different things. rename one of those

Reply:
both should be limited to slip current